config.py

```diff
@@ -81,6 +81,7 @@ def __hash__(self):
 }[ENVIRONMENT_TYPE]
 
 bot_dev_channel_id = {"production": 808138366330994688, "development": 803448149946662923}[ENVIRONMENT_TYPE]
+error_channel_id = {"production": 1017527224540344380, "development": 1017531179664150608}[ENVIRONMENT_TYPE]
```

Comment:
WTF is wrong with GitHub, this code is already in master branch, this is **not** how a PR diff is supposed to be shown after rebasing to master

please close this PR and open a new one, this is impossible to review in this shape IMHO

Reply:
@MatthewBaggins I'm sorry that GitHub was not able to support your workflow for updating this PR to master, if you know any way to review changes between your branch and master without closing this PR and opening a new one, let me know

Reply:
Ok, opened a new one